source/includes/usage-examples/code-snippets/Watch.java

```diff
@@ -0,0 +1,34 @@
+package usage.examples;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.client.ChangeStreamIterable;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.changestream.FullDocument;
+
+public class Watch {
+    public static void main( String[] args ) {
+
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+            MongoCollection<Document> collection = database.getCollection("movies");
+
+            List<Bson> pipeline = Arrays.asList(
+                Aggregates.match(
+                        Filters.in("operationType",
+                                Arrays.asList("insert", "update", "replace"))));
```

Comment:
Nitpick: since the example only performs insert/update/delete, I wonder if we should leave out "replace" to reduce the chance someone might get confused by including it in this situation.